### PR TITLE
update(argo-cd): update argocd helm chart to latest version based on …

### PR DESCRIPTION
**Pull request by jfeldman325**
Updates Argocd to latest version and includes a number of changes to the original values file to support vpcs and ngcp.

### CODE REVIEW

File: charts/argo-cd/values.yaml [hunk 1]
```diff
@@ -1,76 +1,459 @@
...
-
+## Globally shared configuration
 global:
-  skipCrds: true 
-
+  # -- Common labels for the all resources
+  additionalLabels: {}
+    # app: argo-cd 
+  cluster: 
+    ngcp: true
```
Comment (brandonrjacobs) [inline charts/argo-cd/values.yaml:7]: can we default this to false? and keep region ORD1 for standard non NGCP installs.

File: charts/argo-cd/values.yaml [hunk 2]
```diff
@@ -1,76 +1,459 @@
...
-  additionalLabels: {}
-    # app: argo-cd
-
   networkPolicy:
     # -- Create NetworkPolicy objects for all components
     create: false
     # -- Default deny all ingress traffic
     defaultDenyIngress: false
 
+region: ""
```
Comment (brandonrjacobs) [inline charts/argo-cd/values.yaml:79]: duplicate?

Comment (jfeldman325) [reply charts/argo-cd/values.yaml:79]: Nah, just non global override. If a parent chart specifies a certain region you always want the child to be able to modify it. That said I do notice one error which is in the affinity where the retemplating should include a default of that region value.
```
affinity: 
    nodeAffinity:
      requiredDuringSchedulingIgnoredDuringExecution:
        nodeSelectorTerms:
          - matchExpressions:
              - key: topology.kubernetes.io/region
                operator: In
                values:
                  - "{{ default  .Values.global.region .Values.region }}"
              - key: node.coreweave.cloud/class
                operator: In
                values:
                  - cpu
```

File: charts/argo-cd/values.yaml (hunk 1 above)
Comment (jfeldman325) [reply charts/argo-cd/values.yaml:7]: yup that's an error

**Pull request closed without merging**